src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -17,19 +17,20 @@ interface Route {
   total_fees: number;
 }
 
-export interface ConnectorInvoice {
+export interface ConnectorTransaction {
```

Comment:
Renamed because we use this type for both invoices and payments.

Reply:
thanks!